### PR TITLE
Update for local/ DM, given recent model changes

### PR DESCRIPTION
**Pull request by meganokeefe**
Closes https://github.com/CiscoCloud/vaquero/issues/450

Successfully validated + ran local/ SOT on vaquero vagrant + ensured git SOT validates / runs.

Some changes may be needed.

### CODE REVIEW

**Comment by stephenrlouie** [comment]
- core-cloud and core-ignition do not come up with a healthy cluster. 
- the host names also don't get properly set. 

we should run one of the pre `v0.9.0` containers and use the `preview` tool to look at the unattend scripts before and compare to the one's we are generating now

File: local/sites/test-site/env.yml [hunk 1]
```diff
@@ -18,25 +18,22 @@ agent:
 metadata:
   name: detroit-preprod
   kind: pre-prod
-
+  etcd_initial_cluster: etcd-c1=http://10.10.10.10:2380,etcd-c2=http://10.10.10.11:2380,etcd-c3=http://10.10.10.12:2380
```
Comment (stephenrlouie) [inline local/sites/test-site/env.yml]: not always the etcd cluster, will differ depending on what demo you use.

File: local/workflows/ignition-proxy.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+id: ignition-proxy
```
Comment (stephenrlouie) [inline local/workflows/ignition-proxy.yml]: not used

File: local/assets/cloud-config/etcd-proxy.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
+#cloud-config
```
Comment (stephenrlouie) [inline local/assets/cloud-config/etcd-proxy.yml:1]: not currently used, but should be used by the 4th machine in the core-cloud demo `:24`

File: local/assets/ignition/etcd.yml [hunk 4]
```diff
@@ -7,11 +7,11 @@ systemd:
...
             [Service]
-            Environment="ETCD_ADVERTISE_CLIENT_URLS=http://{{.host.ip}}:2379"
-            Environment="ETCD_INITIAL_ADVERTISE_PEER_URLS=http://{{.host.ip}}:2380"
-            Environment="ETCD_LISTEN_CLIENT_URLS=http://{{.host.ip}}:2379,http://127.0.0.1:2379"
+            Environment="ETCD_ADVERTISE_CLIENT_URLS=http://{{.interface.ipv4}}:2379"
+            Environment="ETCD_INITIAL_ADVERTISE_PEER_URLS=http://{{.interface.ipv4}}:2380"
+            Environment="ETCD_LISTEN_CLIENT_URLS=http://{{.interface.ipv4}}:2379,http://127.0.0.1:2379"
             Environment="ETCD_LISTEN_PEER_URLS=http://0.0.0.0:2380"
-            Environment="ETCD_INITIAL_CLUSTER={{ range $i, $node := .env.etcd_ignition_nodes }}{{ if $i }},{{end}}{{$node.name}}=http://{{$node.ip}}:2380{{end}}"
+            Environment="ETCD_INITIAL_CLUSTER={{ range $i, $node := .env.metadata.etcd_ignition_nodes }}{{ if $i }},{{end}}{{$node.name}}=http://{{$node.ip}}:2380{{end}}"
```
Comment (stephenrlouie) [inline local/assets/ignition/etcd.yml:14]: where is the `etcd_ignition_nodes` defined

File: local/sites/test-site/env.yml (hunk 1 above)
Comment (stephenrlouie) [inline local/sites/test-site/env.yml]: not the exact cluster, off by 1 for the core-cloud demo. should go from `.11-.13`

Comment (stephenrlouie) [reply local/sites/test-site/env.yml]: We can define the `etcd_ignition_cluster` here too. I think its a lot cooler to show the loop enumeration in `local/assets/ignition/etcd.yml`, if we can keep that for one of them that will work. It requires each host to define that metadata

File: local/assets/ignition/etcd.yml (hunk 4 above)
Comment (meganokeefe) [reply local/assets/ignition/etcd.yml:14]: in env.yaml 
```
  etcd_ignition_nodes:
    - name: etcd-i1
      ip: 10.10.10.31
    - name: etcd-i2
      ip: 10.10.10.32
    - name: etcd-i3
      ip: 10.10.10.33
```

**Pull request closed without merging**